### PR TITLE
Refactor/replace nusi filter

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
Replace Nusi Filter.

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: core/src/common/components/filter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import React from 'react';
+import { FormBuilder } from 'nusi';
```
Comment (daskyrk) [inline core/src/common/components/filter/index.tsx:15]: Import from relative component path.

File: core/src/common/components/filter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    const formRef = React.useRef<FormInstance>({} as FormInstance);
+
+    React.useImperativeHandle(ref, () => ({
+      form: formRef.current,
+      search,
+    }));
+
+    // 搜索
+    const search = React.useCallback(
+      (_?: React.SyntheticEvent) => {
```
Comment (laojun) [inline core/src/common/components/filter/index.tsx:59]: useless?

File: core/src/common/components/filter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+              } else {
+                formattedValue[name] = curValue;
+              }
+            });
+            if (onSubmit) {
+              onSubmit(formattedValue);
+            }
+          })
+          .catch(({ errorFields }: { errorFields: Array<{ name: any[]; errors: any[] }> }) => {
+            scrollToField(errorFields[0].name);
```
Comment (laojun) [inline core/src/common/components/filter/index.tsx:78]: too much `any` in this file

File: core/src/common/components/filter/index.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  ) => {
+    const formRef = React.useRef<FormInstance>({} as FormInstance);
+
+    React.useImperativeHandle(ref, () => ({
+      form: formRef.current,
+      search,
+    }));
+
+    // 搜索
+    const search = React.useCallback(() => {
```
Comment (daskyrk) [inline core/src/common/components/filter/index.tsx:49]: useless comment

File: core/src/common/components/form-builder/index.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+.erda-form-builder {
+  .erda-form-item-no-label .ant-form-item-label {
+    display: none !important;
+  }
+}
```
Comment (daskyrk) [inline core/src/common/components/form-builder/index.scss:5]: This file is useless.

**Pull request merged**